openmc/data/library.py

```diff
@@ -23,13 +23,16 @@ class DataLibrary(EqualityMixin):
     def __init__(self):
         self.libraries = []
 
-    def get_by_material(self, name):
+    def get_by_material(self, name, xs_type='neutron'):
```

Comment:
Maybe `data_type` would be better (future library types might not necessarily be _just_ cross section data?)

Reply:
Happy new year! That sounds good.